Marlin/Conditionals_LCD.h

```diff
@@ -56,6 +56,10 @@
     #define ENCODER_FEEDRATE_DEADZONE 2
     #define REVERSE_MENU_DIRECTION
 
+  #elif ENABLED(ANET_KEYPAD_LCD) // Leaving for previous config compatability
+
+    #define ZONESTAR_LCD
+
```

Comment:
Either define existing `ANET_KEYPAD_LCD ` if `ENABLED(ZONESTAR_LCD)` is defined [_least attractive option as it just adds config bloat for almost no benefit_] or just accept that you'll use the _ANET_KEYPAD_LCD_ as-is on the Tronxy [preferred option].

Reply:
It seems the 5 button LCD2004 displays are used on more then just the ANET and Tronxy boards.  And in theory, can be used on any board.  I am having a hard time finding verification that these are by zonestar (I replicated this code from someone else), but it seems the other LCD defines are either LCD manufacturer specific or generic entries.  Unless these are by ANET (mine says Tronxy on it), I feel this should have a different define.  Perhaps FIVEBUTTON_KEYPAD or something.
I am seeing references for this type of board via ANet, Tronxy, Zonestar and Hesine.